plugins/catalog/src/components/EntityPageLayout/EntityPageLayout.tsx

```diff
@@ -134,6 +146,19 @@ export const EntityPageLayout = ({ children }: PropsWithChildren<{}>) => {
           <Alert severity="error">{error.toString()}</Alert>
         </Content>
       )}
+
+      {!loading && !error && !entity && (
+        <Content>
+          <WarningPanel title="Entity not found">
+            There was no entity with a matching{' '}
```

Comment:
```suggestion
            There is no entity with a matching{' '}
```

total nitpick: but maybe IS makes more sense since right now as they click the link it can't be found? (not in the past)